css/templates/land-see/page.css

```diff
@@ -351,6 +350,119 @@ amp-img.cover img {
   z-index: 10;
 }
 
-.ampstart-input input[type="email"].user-invalid {
+.land-see-subscribe-form input[type="email"].user-invalid {
   border: 2px solid var(--color-form-invalid);
 }
+
+/* Accordion */
+
+.land-see-accordion section {
+  margin-bottom: 10px;
+  position: relative;
+  overflow: hidden;
+}
+
+.land-see-accordion section > h3 {
+  text-transform: uppercase;
+  font-weight: bold;
+  min-height: 68px;
+  line-height: 68px;
+  outline: none;
+}
+
+.land-see-accordion section > h3:after {
+  content: '';
+  position: absolute;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M6.967 13.932V0M0 6.966h13.932'/%3E%3C/svg%3E");
+  height: 14px;
+  width: 14px;
+  right: 30px;
+  top: 28px;
+  cursor: pointer;
+}
+
+.land-see-accordion section[expanded] > h3:after {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M0 6.966h13.932'/%3E%3C/svg%3E");
+}
+
+.land-see-accordion section:before {
+  content: '';
+  position: absolute;
+  height: 168px;
+  left:0;
+  right: 0;
+  background-color: var(--color-accordion-color-base);
+}
+
+.land-see .i-amphtml-accordion-header { /* reset defaults */
+  cursor: default;
+  background-color: inherit;
+  padding-right: 0;
+  border: none;
+}
+
+.land-see-accordion section.interiors:before {
+  opacity: var(--opacity-100);
+}
+
+.land-see-accordion section.fashion:before {
+  opacity: var(--opacity-77);
+}
+
+.land-see-accordion section.design:before {
+  opacity: var(--opacity-54);
+}
+
+.land-see-accordion section.arts-culture:before {
+  opacity: var(--opacity-30);
+}
+
+/* Post */
+
+.land-see-post-item {
+  max-width: 196px;
+  text-align: left;
+}
+
+.land-see .animate {
+  opacity: 0;
+  transform: translateY(-10px);
+}
+
+.land-see-post-item.wide {
+  max-width: 402px;
+}
+
+.land-see a.land-see-post-category {
+  font: 400 var(--h7)/var(--line-height-1) var(--font-navigation);
+  text-transform: uppercase;
+  color: var(--color-font-accent);
+}
+
+.land-see-post-item h4 {
+  font: 600 var(--h4)/var(--line-height-6) var(--font-secondary);
+  color: var(--color-font-primary);
+}
+
+.land-see-post-item h4 a:hover {
+  opacity: var(--opacity-77);
+}
+
+.land-see-post-item p {
+  font: 400 var(--h6)/var(--line-height-2) var(--font-primary);
+}
+
+/* Instagram/Sale CTAs */
+
+.land-see-cta-link {
```

Comment:
i dont know if the name has to be generic (you take the call). To me CTA sounds very generic , the comment above says its for sale.

Reply:
This naming, as with the accordion, came directly from the design commentary in Basecamp. There are two CTAs (calls to action) on the homepage template:

* Summer Sale (a future PR)
* Instagram

They're very similar, so I wanted to use the same classes for both.